### PR TITLE
Add Python 3.3+ support.

### PR DESCRIPTION
**Pull request by berkerpeksag**

### CODE REVIEW

**Comment by mattrobenolt** [comment]
Can we avoid depending on `six` and just vendorize the few functions we need? I generally don't like to depend on `six` in a simple library. I feel `six` is more valuable in a larger project. I've seen issues with version compatibilities in different versions of `six` which broke the world before in a library.

**Comment by berkerpeksag** [comment]
> Can we avoid depending on six and just vendorize the few functions we need?

Done.

Thanks for the review.

File: MANIFEST.in [hunk 1]
```diff
@@ -1,2 +1,2 @@
-include setup.py README.rst MANIFEST.in LICENSE phabricator/interfaces.json
+include setup.py README.rst MANIFEST.in LICENSE phabricator/interfaces.json setup.cfg tox.ini
```
Comment (mattrobenolt) [inline MANIFEST.in]: We shouldn't need to include setup.cfg or tox.ini. Is there a reason these were added?

Comment (berkerpeksag) [inline MANIFEST.in]: Right, they are not needed. Removed.

**Pull request closed without merging**